gymnasium/envs/mujoco/mujoco_rendering.py

```diff
@@ -50,7 +55,7 @@ def __init__(
         self.viewport = mujoco.MjrRect(0, 0, width, height)
 
         # This goes to specific visualizer
-        self.scn = mujoco.MjvScene(self.model, 1000)
+        self.scn = mujoco.MjvScene(self.model, max_geom or 1000)
```

Comment:
`self.scn = mujoco.MjvScene(self.model, max_geom)`

Reply:
fixed in ede6619.

to avoid breaking I changed all the `max_geom` default values to 1000 (instead of None) in all classes